chef/cookbooks/nova/recipes/config.rb

```diff
@@ -329,7 +316,6 @@
             :keystone_settings => keystone_settings,
             :cinder_insecure => cinder_insecure || keystone_settings['insecure'],
             :ceph_user => ceph_user,
```

Comment:
Ok, so my proposal is to switch off for now this two options in nova.conf and resolve this problem with ephemeral change